anago

```diff
@@ -118,7 +118,7 @@ PROG=${0##*/}
 #+     [--buildversion=ver]      - Override Jenkins check and set a specific
 #+                                 build version
 #+     [--gcrio_repo=repo]       - Specify the GCR repo to use.
```

Comment:
arguably this flag name is also no longer really valid - it should really be `docker_registry` or something like that.

Reply:
PTAL - it seems to make sense now.